FBSnapshotTestCase/Support/SwiftSupport.swift

```diff
@@ -7,11 +7,18 @@
  *
  */
 
+import UIKit
+import XCTest
+
+#if SWIFT_PACKAGE
+@_exported import FBSnapshotTestCaseCore
```

Comment:
I think this is a private Swift feature and not one we should use

Reply:
We need exported the public header of submodules, this case we can use only this in test target

Reply:
why can't people just `import FBSnapshotTestCaseCore` if they need it?

Reply:
in this case `FBSnapshotTestCaseCore` is a private module